cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -66,9 +73,28 @@ def test_appliance_scsi_control_vmware():
         casecomponent: Appliance
         caseimportance: critical
         initialEstimate: 1/4h
-    #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = (conf.cfme_data['basic_info']['cfme_images_url']['cfme_paravirtual_url_format'])\
```

Comment:
This needs to be wrapped with a `pytest.skip` call if this value isn't found. Otherwise the test results in an ERROR with a KeyError.

Kind of personal opinion, but I think this looks a lot cleaner taking advantage of AttrDict. `conf.cfme_data.basic_info.cfme_images_url.cfme_paravirtual_url_format`. Note if you use this syntax, missing keys will result in AttributeError, not KeyError.